player.mli

```diff
@@ -1,23 +1,23 @@
 (** Representation of static player data.
 
     This module represents the data stored in player files.
+    
     @author Keri T. (kmt225)
     Date: 03/25/21
     
 *)
-open Card
 
 (** The abstract type of values representing the player. *)
 type t
 
 (** Raised when player does not have the card in its hand. *)
-exception Card_Not_In_Hand
+exception CardNotInHand
 
 (** [name p] is the name of player [p] *)
 val name : t -> string
 
 (** [player_hand p] is player [p]'s hand of cards *)
-val player_hand : t -> t.hand
+val player_hand : t -> Card.t list
```

Comment:
Are we long abstracting the hand type?

Reply:
I think that might be easier since card.t is already abstracted. I also figured that if hand is abstracted but name isn't the inconsistency may be weird.